### DISCUSSION TITLE
Archway Network Onboarding Proposal

### DISCUSSION THREAD

**Discussion by ethanill** [wormhole-foundation/wormhole#3780] [Gateway]
### Website

https://archway.io/

### Github

https://github.com/archway-network

### Blog

https://blog.archway.io/

### Documentation

https://docs.archway.io/

### Community stats

- 53k Followers on Twitter 
- 2020 Followers on Telegram 
- 33k Users on Discord

### Short Summary

Archway is a Cosmos-native Layer 1 blockchain designed to enable developers to Capture the Value they Create for a network.  Enabling developers to earn has never been more important, as the global blockchain space stands on the precipice of technological commoditization. Archway aims to provide developers with a more equitable distribution of value and opportunities to build self-sustaining dapps, while simultaneously benefiting from the value they bring to the network. By introducing two new modules to the Cosmos SDK, the Protocol distributes a portion of gas fees and inflation, as well as a smart contract premium, to developers.

### Why should Wormhole add this chain?

Archway is a generalized smart contract chain at the heart of the Cosmos ecosystem with simple IBC bridges to local Cosmos chains and the Ethereum ecosystem via Axelar and Gravity bridges. Archway remains a difficult chain to access for Solana users; it’s not impossible, but remains costly with several difficult UX steps.

If Wormhole adds Archway, then the Archway Connect website and other clients like Squid and TFM can offer a faster onboarding path for Solana users. Archway will be able to host a liquidity pool for SOL in the native Astrovault DEX, and enable quick swaps between SOL and ARCH. 

Archway has many native applications and dev teams including; Astrovault, Nomos, ArchMon, Archgregator, Liquid Finance, Architect, Pipapo, etc. Opening up a direct bridge for Wormhole users would allow them access to this ecosystem with ease, and ensure any assets, liquidity and upcoming airdrops here are much more accessible.

Map of zones activity: https://mapofzones.com/home/archway-1/overview?columnKey=ibcVolume&period=24h

### Technology and Features

- CosmosSDK + CometBFT consensus
- Contract Rewards module
- Developer inflation module
- CosmWasm smart contract module
- Account abstraction tools
- Archway CLI
- Multisignature wallet

### Native token details

- 1 billion ARCH max supply
- Governance, network fees, developer rewards, staking

### Technical requirements for running a node

None - Archway will emit messages to Gateway via IBC. Guardians should not have to run anything additional (except for guardians who may run IBC relayers).

### IBC Relaying Strategy

The Forbole team are existing Wormhole Guardians and Archway validators; they have offered to handle IBC relaying. We know of other validator partners that are keen to support too if required - primarily HoodRun

**Comment by Cavalan7** [comment]
sounds great!

**Comment by muriaka** [comment]
Great

**Comment by smithit2000** [comment]
Looks good.

**Comment by Chainode** [comment]
We are ok with the proposal.

**Comment by stakingfacilities-dave** [comment]
Looks good for us. Love to see the leverage of the IBC functionality to onboard more projects.

Staking Facilities supports this proposal.
